### DISCUSSION TITLE
一點關於教育的想法

### DISCUSSION THREAD

**Discussion by Chattenoire** [1101-GS4719-C/Python-Community-GS4719-C#2114] [Ideas]
把個人在臉書上的貼文轉過來，想看看大家有甚麼想法:

「對絕大多數人來說，大學是一個策略性地利用系統達成功利社會目標的場域；大學本身存在的意義被擱置，而系統又以其不可撼動的力量維持著這樣的系譜——我們的社會前所未有地需要一場教育革命；就算今天GPT-o1不出世，這場沉寂太久的教育鬧劇也該改變了。

我期待GPT-o1對於教育系統帶來的衝擊，即使以我目前的年歲可能享受不到它帶來的好處。」

後記: 相信大多數同學都已經或正在目睹系上"內捲"、"分數膨脹"的情況，在國/高中端也有教學內容對於學生在除了升學考試之外毫無助益的情況。我想我的結論不至於激進。

**Comment by yuan3x2** [comment]
我認同「大學是一個策略性地利用系統達成功利社會目標的場域」。然而，大學往往是社會的縮影，或許學生能在此環境中提前體驗和適應並更靈活地銜接未來的職場生活。因此，我認為或許需要一些改革但仍有可以保留的地方。

**Reply by Chattenoire** [reply]
那我想請問同學一件事情:

你認為大學本身的存在意義原來只是、只能是社會的縮影嗎?
或者這樣問好了，我們在發明大學教育制度的時候，想要的是讓學生獲得甚麼——只是單純的社會體驗嗎?

**Reply by yuan3x2** [reply]
我認為大學存在的意義可以是多元的，是一個能讓學生進行更深入學術研究的場域，也可以做為連接職場的橋樑。或許是因為現今學生讀大學的比例相比幾十年前有明顯的增長，因此我認為大學或許不必侷限於學術研究，而非覺得大學只能是社會的縮影唷，希望有回答到你。

**Reply by Karsta0216** [reply]
相比於直接出社會，在大學體驗未來社會的生活相對安全吧。高中畢業之後直接開始工作，白手起家的人一定也有，但我相信他們一定背負著相對比較大的壓力。而我們作為學生，依然享有部分學生犯錯的權利，大學固然是社會的縮影，但造就這樣的結果也是必然的，畢竟未來的我們不可能與社會脫節。自小開始，我們也是在各個家庭裡逐漸的社會化(不管有沒有成功)(成功社會化也很難去定義)。教育需不需要改革我不確定，但是我認為目前的教育並不算非常糟糕。當然，這樣的見解肯定跟每個人的成長環境有關。

**Comment by Teddy111802518** [comment]
基本上對我來說大學是培養學生自我解決問題的開始 ， 雖然相比歐美國家而言慢了許多，但在大學確實需要充分的自主學習，對我而言大學有高度自由，我熱愛這種不被牽絆的學習型態，大部分學生我覺得並沒有理解什麼是學習，我認為讀書不算是所謂的學習，我講的是在台灣教育制度下的讀書，因為我們大部分只會被送，或者去補習班填鴨式教育，但我認為這不是真正的學習，因為學習不止於上課考試，而是對於知識崇高的追求，去理解，去發現，所以興趣多元的我脫離高中之後非常的高興，在實驗室做的超爽，反正我從來就不排斥學習，也就是讀書因為我覺得學習新知不痛苦，痛苦的是背書考試，可能跟我家長不會逼我讀書有關吧

**Comment by poyen0806** [comment]
此篇文章我深有體會，大學是一個每個人用自己的策略去為自己尋求更好的未來，同時大學是一個提供學生學習各種知識的環境，然而現在分數膨脹發生的頻率越來越高，導致所有學生不再在乎自己想學的，而是在乎哪堂課更輕鬆得到高分，雖然這也算是一種策略，但這卻忽視了「學習」。

舉例來說：上個學期系排1的分數是99.5，90.0大概就掉到系排50了🙂

**Comment by Boying12** [comment]
正所謂人生如戲，而教育就只是這部戲的入場卷而已，決定是到馬戲團或國家樂隊，一旦進場後，入場卷的價值驟降，剩的不過是在博取入場卷時，所學到的一點人情世故與社交手段，入場後的表演內容都需要重新再學，教育不過如此，而學習才是一輩子的事
